app/models/comment.rb

```diff
@@ -0,0 +1,12 @@
+class Comment < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  belongs_to :posts
+
+  after_save :update_comments_counter
+
+  private
+
+  def update_comments_counter
+    posts.increment!(:comments_counter)
+  end
```

Comment:
- [ ] [Optional] Great work setting up the relationships and methods in each model. The `after_save` callback function will call the `update_comments_counter` method anytime a comment is made on a post. It would be nice to implement a method that decreases the `comments_counter` anytime a comment is deleted. For this, you can create a method that decreases the comments counter and have the `after_destroy` callback function call it anytime a comment is deleted. You can consider implementing this for all models that have similar methods. 👍